hack/licence-detector/detector/detector.go

```diff
@@ -107,6 +107,23 @@ func doDetectLicences(licenceRegex *regexp.Regexp, classifier *licenseclassifier
 		return nil, nil
 	}
 
+	// this is not an exhaustive list of Elastic-approved licences, but includes all the ones we use to date
+	whitelist := map[string]interface{}{
```

Comment:
Nit: `struct{}` is zero-sized but `interface{}` is not. It's a pretty minor optimization though.